pkg/environment/environment.go

```diff
@@ -169,15 +212,28 @@ func GetDataCatalogServiceAddress() string {
 	return os.Getenv(CatalogConnectorServiceAddressKey)
 }
 
+const printValueStr = "%s set to \"%s\""
```

Comment:
pls put constant at the start of the file

Reply:
Fixed. thanks